### PR TITLE
secrets/ldap: updates API documentation

### PR DESCRIPTION
**Pull request by austingebauer**
This PR updates the API documentation for the LDAP secrets engine. 

Notable changes include:
- Making the sequencing of headings, parameters, and sample requests/responses more consistent
- Adding documentation for service account check-out feature that came from the [AD secrets engine](https://www.vaultproject.io/api-docs/secret/ad#library-management) (slight modifications to the docs)
- Adding the new `last_password` field in the static-creds response
- Adding new configuration parameters

### CODE REVIEW

File: website/content/api-docs/secret/ldap.mdx [hunk 1]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+    "password": "?@09QW0KZ8DSBu3deIu7XLY1NZqzwhozmMAZ6v0IcZJGOjs5GvpVMvOeW7/duls2",
+    "service_account_name": "fizz@example.com"
+  },
+  "wrap_info": null,
+  "warnings": null,
+  "auth": null
+}
+```
+
+## Check-in management
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:713]:
```suggestion
## Check-In Management
```

File: website/content/api-docs/secret/ldap.mdx [hunk 2]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+      "borrower_entity_id": "631256b1-8523-9838-5501-d0a1e2cdad9c"
+    }
+  },
+  "wrap_info": null,
+  "warnings": null,
+  "auth": null
+}
+```
+
+## Check-out management
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:659]:
```suggestion
## Check-Out Management
```

File: website/content/api-docs/secret/ldap.mdx [hunk 3]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+- `ttl` `(duration: "24h", optional)` - The maximum amount of time a single check-out lasts before Vault
+  automatically checks it back in. Defaults to 24 hours. Setting it to zero reflects an unlimited lending period.
+  Uses [duration format strings](/docs/concepts/duration-format).
+- `max_ttl` `(duration: "24h", optional)` - The maximum amount of time a check-out last with renewal before Vault
+  automatically checks it back in. Defaults to 24 hours. Setting it to zero reflects an unlimited lending period.
+  Uses [duration format strings](/docs/concepts/duration-format).
+- `disable_check_in_enforcement` `(bool: false, optional)` - Disable enforcing that service accounts must be
+  checked in by the entity or client token that checked them out. Defaults to false.
+
+### Sample Post Request
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:578]:
```suggestion
### Sample POST Request
```

File: website/content/api-docs/secret/ldap.mdx [hunk 4]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    http://127.0.0.1:8200/v1/ldap/library/accounting-team
+```
+
+### Sample Post Payload
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:588]:
```suggestion
### Sample POST Payload
```

File: website/content/api-docs/secret/ldap.mdx [hunk 5]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+```json
+{
+  "service_account_names": ["fizz@example.com", "buzz@example.com"],
+  "ttl": "10h",
+  "max_ttl": "20h",
+  "disable_check_in_enforcement": false
+}
+```
+
+### Sample Get Response
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:599]:
```suggestion
### Sample GET Response
```

File: website/content/api-docs/secret/ldap.mdx [hunk 6]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+```json
+{
+  "service_account_names": ["fizz@example.com", "buzz@example.com"],
+  "ttl": "10h",
+  "max_ttl": "20h",
+  "disable_check_in_enforcement": false
+}
+```
+
+### Sample List Response
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:610]:
```suggestion
### Sample LIST Response
```

File: website/content/api-docs/secret/ldap.mdx [hunk 7]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+## Library Set Status Check
+
+This endpoint provides the check-out status of service accounts in a library set.
+
+| Method | Path                           |
+| :----- | :----------------------------- |
+| `GET`  | `/ldap/library/:set_name/status` |
+
+### Sample Get Request
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:626]:
```suggestion
### Sample GET Request
```

File: website/content/api-docs/secret/ldap.mdx [hunk 8]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request GET \
+    http://127.0.0.1:8200/v1/ldap/library/accounting-team/status
+```
+
+### Sample Get Response
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:635]:
```suggestion
### Sample GET Response
```

File: website/content/api-docs/secret/ldap.mdx [hunk 9]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+### Parameters
+
+- `name` `(string: "", required)` - The name of the set of service accounts.
+- `ttl` `(duration: "", optional)` - The maximum amount of time a check-out lasts before Vault
+  automatically checks it back in. Setting it to zero reflects an unlimited lending period.
+  Defaults to the set's `ttl`. If the requested `ttl` is higher than the set's, the set's will be used.
+  Uses [duration format strings](/docs/concepts/duration-format).
+
+### Sample Post Request
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:677]:
```suggestion
### Sample POST Request
```

File: website/content/api-docs/secret/ldap.mdx [hunk 10]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    http://127.0.0.1:8200/v1/ldap/library/accounting-team/check-out
+```
+
+### Sample Post Payload
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:687]:
```suggestion
### Sample POST Payload
```

File: website/content/api-docs/secret/ldap.mdx [hunk 11]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+```json
+{
+  "ttl": "1h"
+}
+```
+
+### Sample Post Response
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:695]:
```suggestion
### Sample POST Response
```

File: website/content/api-docs/secret/ldap.mdx [hunk 12]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+| `POST` | `/ldap/library/:set_name/check-in`        |
+| `POST` | `/ldap/library/manage/:set_name/check-in` |
+
+### Parameters
+
+- `name` `(string: "", required)` - The name of the set of service accounts.
+- `service_account_names` `(string: "", or list: [] optional)` - The names of all the service accounts to be
+  checked in. May be omitted if only one is checked out.
+
+### Sample Post Request
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:737]:
```suggestion
### Sample POST Request
```

File: website/content/api-docs/secret/ldap.mdx [hunk 13]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    http://127.0.0.1:8200/v1/ldap/library/accounting-team/check-in
+```
+
+### Sample Post Payload
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:747]:
```suggestion
### Sample POST Payload
```

File: website/content/api-docs/secret/ldap.mdx [hunk 14]
```diff
@@ -536,3 +546,225 @@ The username template cannot use this function.
...
+
+```json
+{
+  "service_account_names": ["fizz@example.com"]
+}
+```
+
+### Sample Post Response
```
Comment (swenson) [inline website/content/api-docs/secret/ldap.mdx:755]:
```suggestion
### Sample POST Response
```

**Review by swenson — APPROVED** [review]
LGTM, some minor nits

**Comment by austingebauer** [comment]
Thanks, @swenson!

**Pull request merged**